nodebuilder/tests/swamp/swamp.go

```diff
@@ -31,6 +34,13 @@ import (
 	"github.com/celestiaorg/celestia-node/nodebuilder/state"
 )
 
+func init() {
+	cfg := sdk.GetConfig()
+	cfg.SetBech32PrefixForAccount(app.Bech32PrefixAccAddr, app.Bech32PrefixAccPub)
+	cfg.SetBech32PrefixForValidator(app.Bech32PrefixValAddr, app.Bech32PrefixValPub)
+	cfg.Seal()
+}
```

Comment:
Let's move this init to the core pkg. This way we always use the right prefix everywhere including swamp. Also if the node is going to be used as a library, then users won't need to do this as well.

Reply:
we are already using it here https://github.com/celestiaorg/celestia-node/blob/311d9dc757f716289978da23913a72fe01c99ede/cmd/celestia/main.go#L18-L21. So, we have to add these lines only for swamp tests